docs/gateway-deployment-via-docker-compose.md

```diff
@@ -0,0 +1,43 @@
+# Deploy Gateway via docker-compose using local servers
+
+* The docker-compose.yml for proxies should work out of the box for usual CoatRack user who seeks to connect with a remote Web Application and a Config Server. 
+* However, as developers you will set up local servers. The problem is that when a docker container accesses `localhost`, it will send a request to itself instead of a host device port. To realize this setup anyhow, perform the following instructions.
+
+
+
+### Starting the Config Server
+
+* Remove the the following attributes from Config Server application.yml and start the Config Server.
```

Comment:
These instructions are not specific for this issue, they are general instructions how to run a local config server in development mode. I would suggest to have a separate issue to create different profiles to run the config server (either with in-memory DB or with persistent Postgres DB).

The related instructions should then be included in the general developer instructions.

- set reasonable

Reply:
In the PR for #152 I already created these two profiles. My intention with this documentation was to say: "As long #122 is not merged, please go the way described in the docs that I provided. This guarantees you to have all information required to test the feature. After it is merged, we will adapt the docs."

Reply:
#152 is a PR, I assume you mean #154?

Anyway, thanks for the explanation. I would then suggest that we mark this PR as draft and merge it after the other issue is solved and after this doc is in the "correct place".

Reply:
Sorry for the confusion. I am talking about PR #152 which will solve issue #122.